crates/ruff_linter/src/rules/pyupgrade/fixes.rs

```diff
@@ -58,7 +58,7 @@ pub(crate) fn remove_import_members(
     }
 
     // Add the remaining content.
-    let slice = locator.after(last_pos);
+    let slice = locator.slice(TextRange::new(last_pos, import_from_stmt.end()));
```

Comment:
Similarly, the final string piece should use the end offset from the statement range.